src/js/model/storage.ts

```diff
@@ -1,30 +1,36 @@
 import { serialize } from 'utils/parser';
 import ApiSettings from 'api/api-settings';
+import type { GenericObject } from 'types/generic.type';
+import type SimpleModel from 'model/simplemodel';
 
 let storage = {
-    removeItem: function() {}
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
+    removeItem: function(itemName: string): void {}
 };
 
 try {
     storage = window.localStorage || storage;
 } catch (e) {/* ignore */}
 
-function Storage(namespace, persistItems) {
-    this.namespace = namespace;
-    this.items = persistItems;
-}
+class Storage {
+    namespace: string;
+    items: any[];
```

Comment:
`items` is a list of keys
```suggestion
    items: string[];
```